benchmark/training/image_classification/test.py

```diff
@@ -12,13 +12,14 @@
 import matplotlib.pyplot as plt
 import pickle
 import tensorflow as tf
-import keras
 from sklearn.metrics import roc_auc_score
 
 import train
 import eval_functions_eembc
 import keras_model
 
+PERF_SAMPLE = True
```

Comment:
Can you add a comment on what True/False means? Are the scripts still working with PERF_SAMPLE = False?

Reply:
It looks like it's a type we can remove, together with line 32.

Reply:
This boolean switches between using the official MLPerf Tiny subset for validation (`True`) or the full CIFAR10 validation set (`False`). I've added a comment to that effect now.